modules/sr/robot3/vision/convert.py

```diff
@@ -22,12 +24,7 @@ class WebotsOrientation(NamedTuple):
 
 
 def rotation_matrix_from_axis_and_angle(orientation: WebotsOrientation) -> Matrix:
```

Comment:
Are we using this anymore?

Reply:
Yes, this is still used in setting up the `FiducialMarker` instance we return. While the orientation of that isn't used here, it will be used to validate that we're looking at the front of a marker (implementation of that will be in a separate PR). This code is validated by the new tests (for single-axis rotations) alongside the direct-to-orientation code-path.

Reply:
> Yes, this is still used in setting up the `FiducialMarker` instance we return. While the orientation of that isn't used here, it will be used to validate that we're looking at the front of a marker (implementation of that will be in a separate PR).

That PR is https://github.com/srobo/competition-simulator/pull/420